apps/web/src/pages/ApplicantDashboardPage/components/FunctionalCommunityListItem.tsx

```diff
@@ -0,0 +1,196 @@
+import { JSX } from "react";
+import { useIntl } from "react-intl";
+import BriefcaseIcon from "@heroicons/react/20/solid/BriefcaseIcon";
+import PresentationChartLineIcon from "@heroicons/react/20/solid/PresentationChartLineIcon";
+import XCircleIcon from "@heroicons/react/20/solid/XCircleIcon";
+import ExclamationTriangleIcon from "@heroicons/react/20/solid/ExclamationTriangleIcon";
+
+import {
+  graphql,
+  PreviewListItemFunctionalCommunityFragment as PreviewListItemFunctionalCommunityFragmentType,
+} from "@gc-digital-talent/graphql";
+import { HeadingLevel, PreviewList } from "@gc-digital-talent/ui";
+
+export const PreviewListItemFunctionalCommunity_Fragment = graphql(
+  /* GraphQL */ `
+    fragment PreviewListItemFunctionalCommunity on CommunityInterest {
+      id
+      jobInterest
+      trainingInterest
+      community {
+        id
+        name {
+          localized
+        }
+        description {
+          localized
+        }
+      }
+    }
+  `,
+);
+
+interface FunctionalCommunityListItemProps {
+  headingAs?: HeadingLevel;
+  functionalCommunityListItemFragment: PreviewListItemFunctionalCommunityFragmentType;
```

Comment:
I think the correct type is:

`query: FragmentType<PreviewListItenmFunctionalCommunity_Fragment>;`

Reply:
https://github.com/GCTC-NTGC/gc-digital-talent/pull/12607/commits/dff254d46efbfb5f6a266186d3eff757e12f4239